fe/fe-core/src/main/java/com/starrocks/analysis/Parameter.java

```diff
@@ -0,0 +1,107 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file is based on code available under the Apache license here:
+//   https://github.com/apache/incubator-doris/blob/master/fe/fe-core/src/main/java/org/apache/doris/analysis/OutFileClause.java
+
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+package com.starrocks.analysis;
+
+import com.starrocks.catalog.Type;
+import com.starrocks.sql.ast.AstVisitor;
+
+public class Parameter extends Expr {
+    private final int slotId;
+
+    private Expr expr;
+
+    private Expr ref;
+
+    public Parameter(int slotId) {
+        this.slotId = slotId;
+    }
+
+    public Parameter(int slotId, Expr expr, Expr ref) {
+        this.slotId = slotId;
+        this.expr = expr;
+        this.ref = ref;
+    }
+
+    public int getSlotId() {
+        return slotId;
+    }
+
+    public Expr getExpr() {
+        return expr;
+    }
+
+    public Expr getRef() {
+        return ref;
+    }
+
+    public void setExpr(Expr expr) {
+        this.expr = expr;
+    }
+
+    public void setRef(Expr ref) {
+        this.ref = ref;
+    }
```

Comment:
what's the purpose of this function and ref field? doesn't seem to find it elsewhere

Reply:
it's not necessary，will delete it